src/App.tsx

```diff
@@ -15,7 +16,7 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-enum SortType {
+export enum SortType {
```

Comment:
why do you export this enum?

Reply:
ohhh.... when something didnt work, i tried to export this enum, dont know why.... end then forgot to delet it...sorry! 😇